.changeset/early-deers-beg.md

```diff
@@ -0,0 +1,16 @@
+---
+'@backstage/plugin-scaffolder-backend-module-github': patch
+---
+
+Fixed bug resulting from missing required owner and repo arguments in `getEnvironmentPublicKey` in action `github:environment:create`.
+
+Adding environment secrets now works as expected.
+
+```diff
+  const publicKeyResponse = await client.rest.actions.getEnvironmentPublicKey({
+    repository_id: repository.data.id,
++     owner,
++     repo,
+      environment_name: name
+  });
+```
```

Comment:
```suggestion
```

Dropping this bit as it's an implementation detail and not too important for the changeset